extensions/gitpod-web/src/extension.ts

```diff
@@ -516,8 +521,20 @@ function getNonce() {
 
 interface PortItem { port: GitpodWorkspacePort; isWebview?: boolean }
 
-function registerPorts(context: GitpodExtensionContext): void {
-	const isPortsViewExperimentEnable = vscode.workspace.getConfiguration('gitpod.experimental.portsView').get<boolean>('enabled');
+async function registerPorts(context: GitpodExtensionContext): Promise<void> {
+
+	const isSaaSGitpod = context.info.getGitpodHost() === 'https://gitpod.io';
```

Comment:
It does not make sense anymore, since sh will use own installation?

Reply:
SH has no configcat SDK key set in their `config.yml`, so `config.json` will respond with empty, get experiment value [will fallback](https://github.com/gitpod-io/gitpod/pull/12756#issuecomment-1241137999) to `undefined` and then default setting in vscode.

Yes, it makes no sense, I'll remove it later